pinot-broker/src/main/java/org/apache/pinot/broker/api/resources/PinotClientRequest.java

```diff
@@ -320,20 +325,21 @@ public void processTimeSeriesQueryEngine(@Suspended AsyncResponse asyncResponse,
     try {
       try (RequestScope requestContext = Tracing.getTracer().createRequestScope()) {
         String queryString = requestCtx.getQueryString();
-        PinotBrokerTimeSeriesResponse response = executeTimeSeriesQuery(language, queryString, Map.of(), requestContext,
+        TimeSeriesBlock timeSeriesBlock = executeTimeSeriesQuery(language, queryString, Map.of(), requestContext,
           makeHttpIdentity(requestCtx), httpHeaders);
+        PinotBrokerTimeSeriesResponse response = PinotBrokerTimeSeriesResponse.fromTimeSeriesBlock(timeSeriesBlock);
```

Comment:
(not introduced in this PR, and actually introduced by me)

We should really change the name of this response object from `PinotBrokerTimeSeriesResponse` to something else.